Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -24,8 +24,15 @@ namespace gdjs {
 
       // Load (or reset)
       if (this._pixiObject === undefined) {
+        const pixiRenderer = runtimeScene
+          .getGame()
+          .getRenderer()
+          .getPIXIRenderer();
+
+        // @ts-ignore - pixi-tilemap types to be added.
+        pixiRenderer.plugins.tilemap = new PIXI.tilemap.TileRenderer();
```

Comment:
same remark here, probably/surely not to be done in a constructor.

Reply:
see https://github.com/pixijs/tilemap/issues/124